### PR TITLE
Ron auth go

### PR DESCRIPTION
**Pull request by ronhorton**
## Issue: 
https://github.com/rancher/qa-tasks/issues/874
 
## Problem
automation doesn't currently have a method to enable/disable and test logins for openldap
 
## Solution

 added openldap extension that enables auth, verifies enabled, disables auth, verifies disabled. 
### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * **Validation (Go Framework)**
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

## QA Testing Considerations

 run opendapauth_test.go
### Regressions Considerations

_TODO_
n/a
Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_
n/a

### CODE REVIEW

File: tests/framework/extensions/authconfigs/openldap/openldapauth_test.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
+package openldapauth
```
Comment (igomez06) [inline tests/framework/extensions/authconfigs/openldap/openldapauth_test.go:1]: Tests don't go in extensions. It's either in validation or integration. But this one in particular should go into validation. It's also missing tags.

**Review by igomez06 — COMMENTED** [review]
Why is this going into v2.6? The tests should not go into extensions.

**Comment by igomez06** [comment]
squash your commits as well.

**Comment by floatingman** [comment]
Could you create a readme explaining how to configure this test?

**Pull request closed without merging**